client/src/components/App.js

```diff
@@ -11,24 +11,43 @@ import ProfileInfoEdit from './profileInfoEdit/ProfileInfoEdit';
 
 import Register from './auth/Register';
 import Login from './auth/Login';
-import Header from './layout/Header'
+import Header from './layout/Header';
+import Sidebar from './layout/Sidebar';
+import { Container, Col, Row } from 'react-bootstrap';
+
+import './layout/Layout.css'
 
 import PublicationPage from './publications/PublicationPage';
+import { Fragment } from 'react';
```

Comment:
Isn't `Fragment` the same as `<></>`? If so, prefer using the latter to maintain consistency.

Reply:
I found that the difference between `Fragment` and `<></>` is that `Fragment` allows key to be passed to `Fragment`, and `<></>` does not support keys or attributes. Not sure if we gonna need to pass attributes in the component later, but should we use `Fragment` in our code for reusability of components? 

Ref: [React - Short Syntax and Keyed Fragments](https://reactjs.org/docs/fragments.html#:~:text=table%3E-,Short%20Syntax,Live,-Demo)